docs/udev-configuration.md

```diff
@@ -41,12 +41,16 @@ To test which devices Akri will discover with a udev rule, you can run the rule
     ```
 1. Reload the udev rules and trigger them.
     ```sh
-    udevadm control --reload
-    udevadm trigger
+    sudo udevadm control --reload
+    sudo udevadm trigger
     ```
-1. List the devices that have been tagged, which Akri will discover.
+1. List the devices that have been tagged, which Akri will discover. Akri will only discover devices with device nodes (devices within the `/dev` directory). These device node paths will be mounted into broker Pods so the brokers can utilize the devices.
     ```sh
-    udevadm trigger --verbose --dry-run --type=devices --tag-match=akri_tag
+    udevadm trigger --verbose --dry-run --type=devices --tag-match=akri_tag | xargs -l bash -c 'if [ -e $0/dev ]; then echo $0/dev; fi'
```

Comment:
If you have the other ones with sudo, should this one use it too?

Reply:
I think only triggering the rules with `sudo udevadm trigger` required sudo while this one doesnt seem to because of the `--dry-run` part and its read only.